Assets/Scripts/MiniGames/Song/SongGameController.cs

```diff
@@ -2,47 +2,122 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
+using System.IO;
+using System.Linq;
 
 public class SongGameController : MonoBehaviour
 {
-    public AudioSource[] musicList;
-    private int songChoiceNum;
-
+	public static bool party_song=false, War_song=false, Sorrow_Song=false, Wisdom_Song=true;
+	public Text victory, failure;
+	private AudioSource[] musicList;
+	public AudioSource[] party_list, War_list, Sorrow_list, Wisdom_list;
```

Comment:
all this code style stuff is minor and it's okay if you don't get it to it, but it'd be nice to update this class to use our standard style while you're in there. i recognize you were just following the conventions that were already used in the file though. no big deal